orbital_core/session.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from aiohttp import ClientSession
+from aiohttp import ServerDisconnectedError, ServerConnectionError
+import backoff
+
+from .utils import tracing
+
+LOG = logging.getLogger(__name__)
+MAX_TRIES = 2
+
+
+@backoff.on_exception(
```

Comment:
is backoff a generic thing we want to add to every http request?

I kind of feel like we should leave that up to the implementer... retries can adversely affect the downstream service, especially if it's not designed to handle double the load.

I'd argue that we should certainly instrument traces, but not include the backoff. thoughts?

Reply:
actually now that I think about it, I think we added this because aiohttp doesn't silently reconnect if the server kills a persistent connection.

can you investigate and figure out if it's the right thing to do her? retry on ServerDisconnected / ServerConnection error?

Reply:
In my opinion let's not add the backoff. It should be handled by implementer and let them decide what to do if error arises.

Reply:
sounds good.